etl/steps/data/grapher/cancer/2024-09-13/diagnosis_routes_survival.py

```diff
@@ -0,0 +1,33 @@
+"""Load a garden dataset and create a grapher dataset."""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load garden dataset.
+    ds_garden = paths.load_dataset("diagnosis_routes_survival")
```

Comment:
It looks like some of the survival rates are above 100%, which surprised me. Perhaps it would be a nice addition to description_key to make a note of why that could be?

Reply:
nicely spotted! I also noticed that and just thought it could be because of the age-standardization but can't see anything specific on this on their website atm! I'll try to dig into this a bit more a bit later!